gemc/5.12/rgc_fall2022.gcard

```diff
@@ -0,0 +1,116 @@
+<!-- GEMC 5.12 -->
+<gcard>
+
+
+
+	
+	<detector name="experiments/clas12/targets/target" factory="TEXT" variation="rgc_summer2022"/>
+
+
+	<!-- central detectors 	-->
+	<detector name="experiments/clas12/bst/bst"               factory="TEXT"/>
+	<detector name="experiments/clas12/micromegas/micromegas" factory="TEXT"/>
+
+	<!--ctof, cnd  -->
+	<detector name="experiments/clas12/ctof/ctof"    factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/ctof/cad/"    factory="CAD"  variation="rga_fall2018"/>
+	<detector name="experiments/clas12/cnd/cnd"      factory="TEXT" variation="rga_fall2018"/>
+
+	<!--high threshold cherenkov -->
+	<detector name="experiments/clas12/htcc/htcc"  factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/htcc/cad/"  factory="CAD"  variation="rga_fall2018"/>
+
+	<!-- magnets -->
+	<detector name="experiments/clas12/magnets/solenoid" factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/magnets/cad/"     factory="CAD" />
+
+	<!-- FT/Beamline configuration -->
+
+	<detector name="experiments/clas12/beamline/cad/"      factory="CAD"  variation="rgc_fall2022"/>
+	<detector name="experiments/clas12/beamline/beamline"  factory="TEXT" variation="rgc_fall2022"/>
+
+	<!-- Detectors in the forward carriage -->
+	<detector name="experiments/clas12/dc/dc"       factory="TEXT"/>
+	<detector name="experiments/clas12/ftof/ftof"   factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/ec/ec"       factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/pcal/pcal"   factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/ltcc/ltcc"   factory="TEXT" variation="rga_fall2018"/>
```

Comment:
RGC should have 4 sectors

Reply:
ltcc changed to rgm_fall2021_H that has 4 sectors